pkg/types/args_test.go

```diff
@@ -0,0 +1,98 @@
+package types_test
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/appc/cni/pkg/types"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+func TestArgs(t *testing.T) {
```

Comment:
See https://github.com/appc/cni/pull/158#discussion_r56532709.
What's the proper solution for generating this file and running the selected test manually?